docs/_quarto.yml

```diff
@@ -125,6 +125,16 @@ website:
         - auto: "how-to/analytics"
         - auto: "how-to/visualization"
         - auto: "how-to/extending"
+
+    - id: examples
+      title: "Examples"
+      style: "docked"
+      collapse-level: 2
+      contents:
+        - auto: "examples/parquet-and-duckdb.qmd"
+        - auto: "examples/clickhouse-hackernews.qmd"
+        - auto: "examples/imdb.qmd"
+
```

Comment:
this isn't showing up at the top like I assume we want, need to add in the navbar left/right thing above

Reply:
Rolled the new docs files into the existing structure.